app/views/projects/samples/metadata/_form.html.erb

```diff
@@ -33,61 +33,48 @@
-            ) %>
             <%= form.label(
-              :key,
+              "key_PLACEHOLDER",
               t("projects.samples.metadata.form.key"),
-              data: {
-                required: true,
-              },
-              class:
-                "keyLabel absolute text-sm text-slate-500 dark:text-slate-400 duration-300 transform -translate-y-4 scale-75 top-2 z-10 origin-[0] bg-white dark:bg-slate-800 px-2 peer-focus:px-2 peer-placeholder-shown:scale-100 peer-placeholder-shown:-translate-y-1/2 peer-placeholder-shown:top-1/2 peer-focus:top-2 peer-focus:scale-75 peer-focus:-translate-y-8 peer-[:not(:placeholder-shown)]:-translate-y-8 rtl:peer-focus:translate-x-1/4 rtl:peer-focus:left-auto start-1 border-none",
+              class: "keyLabel block text-sm font-medium text-slate-700 dark:text-slate-300 mb-1.5",
+              data: { required: "true" }
+            ) %>
+            <%= form.text_field(
+              "key_PLACEHOLDER",
+              id: "key_PLACEHOLDER",
+              class: "keyInput block px-3 py-2 w-full text-sm text-slate-900 bg-white rounded-lg border border-slate-300 appearance-none dark:text-white dark:bg-slate-800 dark:border-slate-600 focus:ring-2 focus:ring-primary-500 focus:border-primary-500 dark:focus:ring-primary-600 dark:focus:border-primary-600",
             ) %>
           </div>
           <div
             id="value[PLACEHOLDER]_value_input"
-            class="
-              relative z-0 w-full group form-field text-slate-500 dark:text-slate-400
-            "
+            class="w-full form-field"
           >
-            <%= form.text_field(
-              :value,
-              id: "value_PLACEHOLDER",
-              class:
-                "valueInput block px-2.5 pb-2.5 pt-4 w-full text-sm text-slate-900 bg-transparent rounded-lg border-1 border-slate-300 appearance-none dark:text-white dark:border-slate-600 peer",
-              placeholder: "",
-            ) %>
             <%= form.label(
-              :value,
+              "value_PLACEHOLDER",
               t("projects.samples.metadata.form.value"),
-              data: {
-                required: true,
-              },
-              class:
-                "valueLabel absolute text-sm text-slate-500 dark:text-slate-400 duration-300 transform -translate-y-4 scale-75 top-2 z-10 origin-[0] bg-white dark:bg-slate-800 px-2 peer-focus:px-2 peer-placeholder-shown:scale-100 peer-placeholder-shown:-translate-y-1/2 peer-placeholder-shown:top-1/2 peer-focus:top-2 peer-focus:scale-75 peer-focus:-translate-y-8 peer-[:not(:placeholder-shown)]:-translate-y-8 rtl:peer-focus:translate-x-1/4 rtl:peer-focus:left-auto start-1 border-none",
+              class: "valueLabel block text-sm font-medium text-slate-700 dark:text-slate-300 mb-1.5",
+              data: { required: "true" }
+            ) %>
+            <%= form.text_field(
+              "value_PLACEHOLDER",
+              id: "value_PLACEHOLDER",
```

Comment:
Suggestion (blocking): If you remove the `id` param then the labels will link properly. This will require `value_` to be replaced with `sample_value_` in `app/javascript/controllers/projects/samples/metadata/create_controller.js` so that the javascript correctly queries the value inputs.